documents/final-submission-document/appendices/personal-reports.tex

```diff
@@ -10,13 +10,13 @@ \subsection{Fabian Hauser}
 I find this quote very fitting for our thesis - not only was it our task to decide on options,
 but also keep questioning them to find the best possible solution.
 
-A tool we used for this were Architectural Design Decisions.
-They supported our path to find good solutions to our problems greatly in my opinion.
-The most surprising turn resulting from an architectural decision during the project was our decision to write an Angular application,
+We used Architectural Design Decisions to find and question possible solutions.
```

Comment:
Architectural Design Decisions uppercase or lowercase? I think I go mad now... :laughing:

Reply:
Changed it to lower case 😆